V/VMEC/build_tarballs.jl

```diff
@@ -1,43 +1,106 @@
+           ArchiveSource("https://gitlab.com/wistell/VMEC2000/-/archive/v$(version).tar",
+                         "5178816df28db77cb0aed82d8d3743836431da5b96e38013031e34fdd1d2675b"),
 ]
 
 script = raw"""
-mkdir -p ${libdir}
-cd ${WORKSPACE}/srcdir/VMEC2000*
-make USE_MKL=1
-install -m755 lib/libvmec_mkl.so ${libdir}
-make USE_MKL=1 vmec_clean ; make USE_MKL=0
-install -m755 lib/libvmec_openblas.so ${libdir}
-make USE_MKL=0 vmec_clean
+cd $WORKSPACE/srcdir/VMEC*
+if [[ ${target} == *mingw* ]]; then
+  sed "s/LT_INIT/LT_INIT(win32-dll)/" configure.ac | cat > configure.ac.2
+  mv configure.ac.2 configure.ac
+  ./autogen.sh
+  ./configure CC=gcc FC=gfortran F77=gfortran --with-mkl --enable-tbb --host=$target --target=$target --prefix=$prefix
+  # Deal with the issue the gcc doesn't accept -no-undefined at configure step
+  sed "s/AM_LDFLAGS =/AM_LDFLAGS =-no-undefined/" Makefile | cat > Makefile.2
+  mv Makefile.2 Makefile
+  make && make install && make clean
+
+  ./configure CC=gcc FC=gfortran F77=gfortran --with-mkl --host=$target --target=$target --prefix=$prefix
+  # Deal with the issue the gcc doesn't accept -no-undefined at configure step
+  sed "s/AM_LDFLAGS =/AM_LDFLAGS =-no-undefined/" Makefile | cat > Makefile.2
+  mv Makefile.2 Makefile
+  make && make install && make clean
+
+  ./configure CC=gcc FC=gfortran F77=gfortran --host=$target --target=$target --prefix=$prefix
+  # Deal with the issue the gcc doesn't accept -no-undefined at configure step
+  sed "s/AM_LDFLAGS =/AM_LDFLAGS =-no-undefined/" Makefile | cat > Makefile.2
+  mv Makefile.2 Makefile
+  make && make install && make clean
+
+else
+  # Need to apply a fix on apple-darwin* to add missing symlinks for older libmbed* library versions
+  if [[ ${target} == *apple-darwin* ]]; then
+    # Apply libmbed fixes
+    libmbed_path=$(find /workspace/${bb_full_target}/artifacts -name 'libmbedtls.a')
+    artifact_id=$(echo $libmbed_path | grep -o -e "artifacts/[a-z0-9]*" | grep -o -e "/[a-z0-9]*" | grep -o -e "[a-z0-9]*")
+    mbed_version=$(find ${WORKSPACE}/${bb_full_target}/artifacts/${artifact_id}/lib/ -name "libmbedtls.[0-9]*.[0-9]*.[0-9]*.${dlext}" | grep -o -e "[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*")
+    # mbed_version has a leading dot
+    cd ${WORKSPACE}/${bb_full_path}/destdir/lib
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedtls${mbed_version}.${dlext} libmbedtls.13.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedtls${mbed_version}.${dlext} libmbedtls.12.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedcrypto${mbed_version}.${dlext} libmbedcrypto.5.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedcrypto${mbed_version}.${dlext} libmbedcrypto.3.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedcrypto${mbed_version}.${dlext} libmbedcrypto.0.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedx509${mbed_version}.${dlext} libmbedx509.0.${dlext}
+    cd ${WORKSPACE}/srcdir/VMEC*
+  fi
```

Comment:
What's the problem with mbedtls?

Reply:
On `x86_64-apple-darwin*`, during the linking stage the linker looks for older versions of `libmbedtls`, `libmbedcrypto` and `libmbedx509`.  Creating these links allowed the linking stage to pass on `x86_64-apple-darwin*`, so this was my work around.  This is the first time I've really delved into using autotools and trying to cross-compile, so I am probably making some errors.

Reply:
I'm not sure what's the problem exactly (I don't see it), but this is very likely _**not**_ the right thing to do.

Reply:
Thanks Mosé for the help, I `libmbed` issue was due to the version being too new, so I've specified a slightly older version.  Thanks for bearing with me while I figure this out!